frontend-nanodegree-arcade-game/js/app.js

```diff
@@ -136,6 +143,123 @@ const Player = class{
     }
 };
 
+//Class Gem
+const Gem = class{
+    constructor(){
```

Comment:
const Obstacle = class{  <-- space between the class and {
    constructor(){ <-- same here
        this.sprite = 'images/Rock.png';
        this.x = randomPosition(xPositions);
        this.y = randomPosition(yPositions);
        this.width = 101;
    }

Reply:
I see the spacing problems. I'm fixing them right now.